### PR TITLE
Value sets for the Simple Observation profile

### PR DESCRIPTION
**Pull request by Gokulsignal**
@enjoysparkling , Hi Matt, Kindly review the pull request.

### CODE REVIEW

File: input/vocabulary/valueset-reference-range-applies-to.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-reference-range-applies-to.json:1]: This one is missing the expansion values from the v3-race code system.  Include it, even though this is an example binding.

File: input/vocabulary/valueset-reference-range-meaning.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-reference-range-meaning.json:1]: There is an implicit hierarchy on the http://hl7.org/fhir/R4/valueset-referencerange-meaning.html, but it's not represented here.  If we don't think Devs will use it, we can keep it like this.  
Double-check with UI/Devs to see if they will be using this value set and if they will want the hierarchy for display purposes.

File: input/vocabulary/valueset-us-core-simple-observation-category.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-us-core-simple-observation-category.json:1]: We are missing the category (default) valueset expansion.  Include it here.  
http://hl7.org/fhir/R4/valueset-observation-category.html

File: input/vocabulary/valuset-data-absent-reason.json
Comment (enjoysparkling) [inline input/vocabulary/valuset-data-absent-reason.json:1]: There is an implicit hierarchy on the http://hl7.org/fhir/R4/valueset-data-absent-reason.html, but it's not represented here. If we don't think Devs will use it, we can keep it like this.
Double-check with UI/Devs to see if they will be using this value set and if they will want the hierarchy for display purposes.

File: input/vocabulary/valuset-observation-status.json
Comment (enjoysparkling) [inline input/vocabulary/valuset-observation-status.json:1]: There is an implicit hierarchy on the http://hl7.org/fhir/R4/valueset-observation-status.html, but it's not represented here. If we don't think Devs will use it, we can keep it like this.
Double-check with UI/Devs to see if they will be using this value set and if they will want the hierarchy for display purposes.

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
See comments on files.  
1. Confirm if we need to represent an expansion JSON hierarchy for a few of these
2. add category valueset

File: input/vocabulary/valueset-reference-range-applies-to.json
Comment (Gokulsignal) [reply input/vocabulary/valueset-reference-range-applies-to.json:1]: Values are expanded from v3-race code system. 

https://github.com/enjoysparkling/signal-mso-fhir-profiles/commit/5d446702f0a9bcf68f2d0aea8812730275b1085b

Kindly review

File: input/vocabulary/valueset-reference-range-meaning.json
Comment (Gokulsignal) [reply input/vocabulary/valueset-reference-range-meaning.json:1]: Checked with Dev team. At this point, there are no requirements in their screens to use this ValueSet. May be there will be requirements at future.

File: input/vocabulary/valuset-data-absent-reason.json
Comment (Gokulsignal) [reply input/vocabulary/valuset-data-absent-reason.json:1]: Checked with Dev team. At this point, there are no requirements in their screens to use this ValueSet. May be there will be requirements at future.

File: input/vocabulary/valuset-observation-status.json
Comment (Gokulsignal) [reply input/vocabulary/valuset-observation-status.json:1]: Checked with Dev team. At this point, there are no requirements in their screens to use this ValueSet. May be there will be requirements at future.

**Review by enjoysparkling — APPROVED** [review]

**Pull request merged**